scripts/matchSpecialPsyToUniversities.js

```diff
@@ -6,45 +6,71 @@ const psyToUni = require('./psyToUni');
  * Update a list of special psy from a excel sheet to match them with a university
  * can be run and rerun again
  */
-const run = async () => {
-  console.log('Match psychologists to universities...');
-  const universities = await dbUniversities.getUniversities();
-  console.log(`${universities.length} universities`);
-  const psychologists = await dbPsychologists.getPsychologistsDeclaredUniversity();
+const run = async (dryRun) => {
+  console.log('Match special psychologists to universities...');
+  if (dryRun) {
+    console.log('WARNING: dry-run mode on! (no changes will be applied)');
+  }
 
-  const statsNoEmailFound = [];
-  const statsConflictingDeclaredUniversity = [];
+  try {
+    const universities = await dbUniversities.getUniversities();
+    const psyFromDb = await dbPsychologists.getAcceptedPsychologists([
+      'personalEmail',
+      'dossierNumber',
+      'assignedUniversityId',
+    ]);
 
-  const specialCasePsychologists = psychologists.filter((psy) => psyToUni[psy.personalEmail] !== undefined);
-  const needToWait = specialCasePsychologists
+    const statsAssignmentDone = [];
+    const statsNoUniFound = [];
+    const statsNoChange = [];
+    const psyFoundInDb = psyFromDb.filter((psy) => psyToUni[psy.personalEmail] !== undefined);
+    const emailPsyFromDb = psyFromDb.map((p) => p.personalEmail);
+    const statsNoPsyFound = Object.keys(psyToUni).filter((psy) => emailPsyFromDb.includes(psy) === false);
+
+    const needToWait = psyFoundInDb
     .map((psy) => {
-      const psychologist = { ...psy };
       const universityIdToAssign = dbUniversities.getUniversityId(universities, psyToUni[psy.personalEmail]);
```

Comment:
not part of this pr but this has nothing to do in the db !!!
and grouping them by universityName once would be more efficient